pkg/sql/schemachanger/scbuild/internal/scbuildstmt/drop_function.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2023 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package scbuildstmt
+
+import (
+	"github.com/cockroachdb/cockroach/pkg/sql/schemachanger/scerrors"
+	"github.com/cockroachdb/cockroach/pkg/sql/schemachanger/scpb"
+	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
+)
+
+func DropFunction(b BuildCtx, n *tree.DropFunction) {
+	if n.DropBehavior == tree.DropCascade {
+		// TODO(chengxiong): remove this when we allow UDF usage.
+		panic(scerrors.NotImplementedErrorf(n, "cascade dropping functions"))
+	}
+	for _, f := range n.Functions {
+		elts := b.ResolveUDF(&f, ResolveParams{
+			IsExistenceOptional: n.IfExists,
```

Comment:
shouldn't here be another `requiredPrivilege : DROP` for `b.ResolveUDF`?

Reply:
Function only has `EXECUTE` privilege. Only function owner can drop the function or perform any schema changes on function. So we have `b.mustOwn(fnID)` in `ResolveUDF` to validate that